tools/bowtie2.wdl

```diff
@@ -0,0 +1,410 @@
+            bowtie2_option: "I|minins",
+        }
+        max_fragment_len: {
+            description: "The maximum fragment length for valid paired-end alignments.",
+            bowtie2_option: "X|maxins",
+        }
+        no_mixed: "By default, when bowtie2 cannot find a concordant or discordant alignment for a pair, it then tries to find alignments for the individual mates. This option disables that behavior."
+        no_discordant: "By default, bowtie2 looks for discordant alignments if it cannot find any concordant alignments. A discordant alignment is an alignment where both mates align uniquely, but that does not satisfy the paired-end constraints. This option disables that behavior."
+        dovetail: {
+            description: "If the mates 'dovetail', that is if one mate alignment extends past the beginning of the other such that the wrong mate begins upstream, consider that to be concordant.",
+            external_help: "https://bowtie-bio.sourceforge.net/bowtie2/manual.shtml#mates-can-overlap-contain-or-dovetail-each-other",
+        }
+        no_contain: {
+            description: "If one mate alignment contains the other, consider that to be non-concordant.",
+            external_help: "https://bowtie-bio.sourceforge.net/bowtie2/manual.shtml#mates-can-overlap-contain-or-dovetail-each-other",
+        }
+        no_overlap: {
+            description: "If one mate alignment overlaps the other at all, consider that to be non-concordant.",
+            external_help: "https://bowtie-bio.sourceforge.net/bowtie2/manual.shtml#mates-can-overlap-contain-or-dovetail-each-other",
+        }
+        time: "Print the wall-clock time required to load the index files and align the reads to STDERR."
+        write_unpaired_unaligned: "Write unpaired reads that didn't align to a file in gzipped FASTQ format. These reads correspond to the SAM records with the FLAGS 0x4 bit set and neither the 0x40 nor 0x80 bits set."
+        write_unpaired_aligned: "Write unpaired reads that aligned at least once to a file in gzipped FASTQ format. These reads correspond to the SAM records with the FLAGS 0x4, 0x40, and 0x80 bits unset."
+        write_paired_discordant: "Write pairs that didn't align concordantly to a file in gzipped FASTQ format. These reads correspond to the SAM records with the FLAGS 0x4 bit set and either the 0x40 or 0x80 bit set (depending on whether it's mate #1 or #2)."
+        write_paired_concordant: "Write pairs that aligned concordantly at least once to a file in gzipped FASTQ format. These reads correspond to the SAM records with the FLAGS 0x4 bit unset and either the 0x40 or 0x80 bit set (depending on whether it's mate #1 or #2)."
+        quiet: "Print nothing besides alignments and serious errors."
+        metrics_file: "Write bowtie2 alignment metrics to file. Useful for debugging."
+        metrics_stderr: "Write  bowtie2 alignment metrics to STDERR. Not mutually exclusive with `metrics_file`."
```

Comment:
```suggestion
        metrics_stderr: "Write bowtie2 alignment metrics to STDERR. Not mutually exclusive with `metrics_file`."
```
Why would someone enable this in addition to `metrics_file`?

Reply:
The only reason I can see is to write a file and stream to the screen simultaneously, though there would be better ways to handle that (pipe `STDERR` through `tee` for example).